### PR TITLE
Restart application after reset settings

### LINKED ISSUE #440: Restart program automatically after reset settings.
The message we give out now says that we will do this, but it doesn't happen. Maybe we should change it in the meantime.

### PR DESCRIPTION
**Pull request by patrickelectric**
Tested in:
- [x] Linux
- [x] Windows
- [x] Mac

Fix #440

### CODE REVIEW

**Comment by patrickelectric** [comment]
Tested and working, downloads [available here](https://github.com/bluerobotics/ping-viewer/releases/tag/continuous-restart).

**Comment by jaxxzer** [comment]
Thanks @patrickelectric. Do you have any reference?

**Comment by patrickelectric** [comment]
@jaxxzer reference for what exactly ?

File: src/util/util.cpp [hunk 1]
```diff
@@ -72,6 +72,17 @@ void Util::update(QtCharts::QAbstractSeries* series, const QVector<double>& poin
     xySeries->replace(realPoints);
 }
 
+void Util::restartApplication()
```
Comment (jaxxzer) [inline src/util/util.cpp]: I mean did you use some example to create this, or did you create the solution yourself?

Comment (patrickelectric) [reply src/util/util.cpp]: Added.

**Pull request merged**